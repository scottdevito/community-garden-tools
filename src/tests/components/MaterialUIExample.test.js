import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { shallow, mount, render } from 'enzyme';

import MaterialUIExample from '../../components/MaterialUIExample';

// Snapshot test
it('renders correctly', () => {
  const tree = renderer.create(<MaterialUIExample />).toJSON();
  expect(tree).toMatchSnapshot();
});
