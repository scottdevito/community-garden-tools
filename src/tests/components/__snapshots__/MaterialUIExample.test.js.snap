// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-EHOje enDNEB"
>
  <h1
    className="sc-bZQynM bOrPFB"
  >
    Material UI
  </h1>
  <ul>
    <li>
      Install Material UI
    </li>
    <li>
      Create .MUI.js component
    </li>
    <li>
      Import .mui.js component into render method
    </li>
  </ul>
  <div
    className="sc-ifAKCX eVrQyi"
  >
    <h2>
      Resources:
    </h2>
    <a
      href="https://material-ui.com/demos/selects"
    >
      https://material-ui.com/demos/selects
    </a>
  </div>
</div>
`;
